src/Telegram.Bot/Types/Enums/MessageType.cs

```diff
@@ -72,6 +73,7 @@ public enum MessageType
         /// <summary>
         /// The <see cref="Message"/> contains a <see cref="Types.VideoNote"/>
         /// </summary>
+        [EnumMember(Value = "video_note")]
```

Comment:
By the way, this thing might be a breaking change. Someone might serialize update and/or message objects as is and store somewhere. And then rely on this values being in camelCase.